vue/configurator/src/views/MarketplaceView.vue

```diff
@@ -144,6 +175,17 @@ export default class MarketplaceView extends Vue {
     const { detail } = event;
     console.log("Marketplace Widget Injected", detail);
   }
+
+  getSystemPreference(): string {
+    if (
+      window.matchMedia &&
+      window.matchMedia("(prefers-color-scheme: dark)").matches
+    ) {
+      return "manifold-scheme-dark";
+    } else {
+      return "manifold-scheme-light";
+    }
+  }
```

Comment:
is this function used anywhere?

Reply:
Removed! I was exploring OS-preference with different approaches, decided to not use this